tests/func/test_live.py

```diff
@@ -1,12 +1,13 @@
 import os
+from contextlib import suppress
 from copy import deepcopy
 from textwrap import dedent
 
 import pytest
 from funcy import first
 
 from dvc import stage as stage_module
-from dvc.exceptions import MetricsError
+from dvc.exceptions import MetricDoesNotExistError, MetricsError
 
 LIVE_SCRITP = dedent(
```

Comment:
Can the spelling be fixed here and where it's used below as long as other changes are being made?

```suggestion
LIVE_SCRIPT = dedent(
```